receiver/googlecloudmonitoringreceiver/README.md

```diff
@@ -0,0 +1,61 @@
+| [Code Owners](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/CONTRIBUTING.md#becoming-a-code-owner)    | [@dashpole](https://www.github.com/dashpole), [@TylerHelmuth](https://www.github.com/TylerHelmuth), [@abhishek-at-cloudwerx](https://www.github.com/abhishek-at-cloudwerx) |
+
+[development]: https://github.com/open-telemetry/opentelemetry-collector#development
+[contrib]: https://github.com/open-telemetry/opentelemetry-collector-releases/tree/main/distributions/otelcol-contrib
+<!-- end autogenerated section -->
+
+The primary objective of the Google Cloud Monitoring Receiver is to gather time series metrics data from all Google services and convert this data into a pipeline format that facilitates further use.
+
+This receiver gets GCP (Google Clout Platform) metrics from [GCP Monitoring REST API] via the [Google SDK for GCP Metrics] and then convert those timeseries data into OTel Format [Pipeline Data].
+
+[GCP Monitoring REST API]: https://cloud.google.com/monitoring/api/ref_v3/rest/v3/projects.timeSeries/list
+[Google SDK for GCP Metrics]: https://pkg.go.dev/cloud.google.com/go/monitoring/apiv3
+[Pipeline Data]: https://pkg.go.dev/go.opentelemetry.io/collector/pdata
+
+## Configuration
+The following configuration options are supported:
+
+```yaml
+receivers:
+  googlecloudmonitoring:
+    collection_interval: 120s
+    project_id: my-project-id
+    metrics_list:
+      - metric_name: "compute.googleapis.com/instance/cpu/usage_time"
+        delay: 60s
+      - metric_name: "connectors.googleapis.com/flex/instance/cpu/usage_time"
+        delay: 60s
+```
+
+- `collection_interval` (Optional): The interval at which metrics are collected. Default is 60s.
+- `initial_delay` (default = `1s`): defines how long this receiver waits before starting.
+- `timeout`: (default = `1m`) The timeout of running commands against the GCP Monitoring REST API.
+- `project_id` (Required): The GCP project ID.
+- `metrics_list` (Required): A list of services metrics to monitor.
+
+Each single metric can have the following configuration:
+
+- `metric_name` (Required): The specific metric name to collect.
+- `delay` (Optional): The delay before starting the collection of metrics for this service. Default is 0s.
+
+## Exporting Google Application Credentials
+
+To authenticate with Google Cloud services, set the GOOGLE_APPLICATION_CREDENTIALS environment variable to the path of your service account key file (google_application_credentials.json). This file contains the necessary credentials for accessing Google Cloud APIs securely.
```

Comment:
I would recommend linking to https://cloud.google.com/docs/authentication/application-default-credentials.  Note that workloads on GCP will use the service accounts credentials without setting the environment variable.

Reply:
Updated